services/QuillLMS/app/controllers/subscriptions_controller.rb

```diff
@@ -74,9 +74,12 @@ def destroy
     if current_subscription
       @subscription_status_obj = current_subscription
       expired = false
-    else
+    elsif current_user.last_expired_subscription
       @subscription_status_obj = current_user.last_expired_subscription
       expired = true
+    else
+      @subscription_status = nil
+      return
```

Comment:
This was causing a bug where users who had never had a subscription were seeing "your subscription is expired".